addons/im_livechat/static/src/core/web/livechat_channel_tag_model.js

```diff
@@ -0,0 +1,12 @@
+import { Record } from "@mail/core/common/record";
+
+export class LivechatChannelTag extends Record {
+    static _name = "im_livechat.channel.tag";
+    static id = "id";
+
+    /** @type {number} */
+    id;
+    /** @type {string} */
+    name;
+}
+LivechatChannelTag.register();
```

Comment:
I know it's optional, but you might want to create the relation to threads with the inverse? Models are also documentation for me, I like it when everything is explicit.